types/vscode-webview/index.d.ts

```diff
@@ -0,0 +1,50 @@
+// Type definitions for non-npm package vscode-webview 1.57
+// Project: https://code.visualstudio.com/api/extension-guides/webview
+// Definitions by: Matt Bierner <https://github.com/mjbvz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.0
+
+/**
+ * API exposed to webviews.
+ *
+ * @template StateType Type of the persisted state stored for the webview.
+ */
+export interface WebviewApi<StateType> {
+    /**
+     * Post a message to the owner of the webview.
+     *
+     * @param message Data to post. Must be JSON serializable.
+     */
+    postMessage(message: any): void;
```

Comment:
just a style comment: I prefer to take parameters as `unknown` instead of any. Aside from preventing linters from yelling at me if I have it configured to ban `any`, it means that classes which derive the interface and copy from the interface definition don't automatically get the virality of `any`